dmalloc_argv.c

```diff
@@ -820,10 +820,7 @@ static	void	usage_long(const argv_t *args)
       col_c++;
     }
     else {
-      if (arg_p->ar_short_arg == '\0') {
```

Comment:
This changes the logic here right?  If it is \0 then it's going to call into the else block below.

Reply:
Line 818 already check '\0'